src/System/Wlog/IOLogger.hs

```diff
@@ -294,10 +258,23 @@ setLevel p = lLevel .~ Just p
 clearLevel :: Logger -> Logger
 clearLevel = lLevel .~ Nothing
 
+-- | Set severity for given logger. By default parent's severity is used.
+setSeverity :: MonadIO m => LoggerName -> Severity -> m ()
+setSeverity name =
+    liftIO . updateGlobalLogger name . setLevel
```

Comment:
You don't need `liftIO` here because `updateGlobalLogger` is already lifted. And you can implement this function in one line, no need in moving function body to separate line.